app/models/category.rb

```diff
@@ -0,0 +1,3 @@
+class Category < ApplicationRecord
+  has_many :cashflows
+end
```

Comment:
- [ ] Let's add model validations to avoid entering bad data.

Reply:
well resolved! ✔️